app/services/users.js

```diff
@@ -0,0 +1,34 @@
+const logger = require('../logger');
+const { conflictError, databaseError } = require('../errors');
+const { createHash } = require('../helpers/bcrypt');
+const { User } = require('../models');
+const {
+  validationMessages: { EMAIL_ALREADY_ERROR }
+} = require('../constants');
+
+exports.getUserByEmail = async email => {
+  try {
+    const result = await User.findOne({ where: { email } });
```

Comment:
You aren't using the result later, for that reason you can return it directly:
```
return await User.findOne({ where: { email } });
```

Reply:
@yovanny-wolox sorry, but I couldn’t see where are you using this method.

Reply:
Oops, I'm sorry. I created the method but finally did not use it and forgot removed. Fixed!